core/lib/config/src/configs/da_dispatcher.rs

```diff
@@ -14,6 +15,10 @@ pub struct DADispatcherConfig {
     pub max_rows_to_dispatch: Option<u32>,
     /// The maximum number of retries for the dispatch of a blob.
     pub max_retries: Option<u16>,
+    /// Enable on-chain verification of the data availability.
+    // TODO: run a verification task to check if the L1 contract expects the inclusion proofs.
+    // Maybe even remove this value and use the L1 as a source of truth.
+    pub enable_onchain_verification: Option<bool>,
```

Comment:
I think `enable_onchain_verification` name and comment are a bit misleading. On-chain verification is fully controlled by L1 contract and server can't enable or disable it, instead variable controls if dummy inclusion data is used or real one is awaited. Maybe `use_dummy_inclusion_data` or `use_blank_inclusion_data` would be a better name?

Reply:
good point, changed it to `use_dummy_inclusion_data`